README.rst

```diff
@@ -20,49 +18,83 @@ The `auth_backends` package can be installed from PyPI using pip::
++----------------------------------------------+---------------------------------------------------------------------------------------------+
+| SOCIAL_AUTH_EDX_OIDC_SECRET                  | OAuth/OpenID Connect client secret                                                          |
++----------------------------------------------+---------------------------------------------------------------------------------------------+
+| SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY | Identity token decryption key (same value as the client secret for edX OpenID Connect)      |
++----------------------------------------------+---------------------------------------------------------------------------------------------+
+| SOCIAL_AUTH_EDX_OIDC_URL_ROOT                | OAuth/OpenID Connect provider root (e.g. https://courses.stage.edx.org/oauth2)              |
++----------------------------------------------+---------------------------------------------------------------------------------------------+
+| SOCIAL_AUTH_EDX_OIDC_LOGOUT_URL              | OAuth/OpenID Connect provider's logout page URL (e.g. https://courses.stage.edx.org/logout) |
++----------------------------------------------+---------------------------------------------------------------------------------------------+
 
-Set these to the correct values for your OAuth2/OpenID Connect provider. ``SOCIAL_AUTH_EDX_OIDC_ID_TOKEN_DECRYPTION_KEY``
-should be the same as ``SOCIAL_AUTH_EDX_OIDC_SECRET``. Set ``EXTRA_SCOPE`` equal to a list of scope strings to request
-additional information from the edX OAuth2 provider at the moment of authentication (e.g., provide course permissions bits
-to get a full list of courses).
 
-Views
------
+If your application requires additional user data in the identity token, you can specify additional claims by defining
+the ``EXTRA_SCOPE`` setting. For example, if you wish to have a claim named `preferred_language`, you would include
+the following in your settings:
 
-This package includes a login view, ``EdxOpenIdConnectLoginView``, that utilizes edX's implementation of OpenID Connect.
-This should be used for all microservices/IDAs.
+.. code-block:: python
 
-``LogoutRedirectBaseView`` is also included. It should be overridden, and customized. See the docstring for details.
+    EXTRA_SCOPE = ['preferred_language']
 
-urls.py
--------
+Assuming the identity provider knows how to process that scope, the associated claim data will be included in the
+identity token returned during authentication. Note that these scopes/claims are dependent on the identity provider
+being used. The ``EdXOpenIdConnect`` backend is configured to be used by all edX services out-of-the-box.
+
+There is an additional setting, ``COURSE_PERMISSIONS_CLAIMS``, used by
+`edx-analytics-dashboard <https://github.com/edx/edx-analytics-dashboard>`_. Its usage is not recommended due to
```

Comment:
I'm not sure this is totally accurate.  The initial performance concerns were in querying courses that the user had access to and was slow prior to using the course overviews table to provide the list of courses.